### ISSUE TITLE
Autoware on Jetson Xavier

### ISSUE THREAD

**Issue by kennedywai** [autowarefoundation/autoware_ai#293]
Anyone who has plans to run Autoware on Jetson Xavier Developer Kit? We could work together on this.

**Comment by IndefiniteBen** [comment]
Related to autowarefoundation/autoware_ai#406

**Comment by IndefiniteBen** [comment]
I haven't actually done this, but my research on how to run Autoware on a TX2 with JetPack 4.2 lead me to the same conclusion that only ROS Melodic will work. Based on what I've found, the issues that were blocking Melodic support are fixed (see my comment on autowarefoundation/autoware_ai#406) but it hasn't been moved to release. If you look at [autowarefoundation/autoware/tree/feature/melodic](https://github.com/autowarefoundation/autoware/tree/feature/melodic) you can see the branch with these changes. Pull that branch and follow the build from source instructions? 

tl;dr
Pull [autowarefoundation/autoware/tree/feature/melodic](https://github.com/autowarefoundation/autoware/tree/feature/melodic)
Follow these [instructions](https://github.com/autowarefoundation/autoware/wiki/Source-Build)
